packages/ui/src/showcases/Transfer.stories.tsx

```diff
@@ -0,0 +1,375 @@
+import React, { useEffect, useState } from 'react'
+import { BrowserRouter } from 'react-router-dom'
+
+import { AddressInput } from '../AddressInput'
+import { BaseInput } from '../BaseInput'
+import { Button } from '../Button'
+import { Icon } from '../Icon'
+import {
+  BoxLayout,
+  CardLayout,
+  ColumnLayout,
+  GridLayout,
+  PageLayout,
+  StackLayout,
+  usePageLayout,
+} from '../Layouts'
+import { Menu, MenuItem } from '../Menu'
+import { ColumnData, Table } from '../Table'
+import { Text } from '../Text'
+
+export default {
+  title: 'Showcases/Transfer',
+}
+
+export const Overview = args => {
+  const nav = usePageLayout()
+  const { setHeader } = nav
+
+  useEffect(() => {
+    if (args.header) {
+      setHeader(
+        <PageLayout.Header
+          navigationItems={[
+            { to: '/', label: 'Dashboard' },
+            { to: '/deposit', label: 'Deposit & Borrow' },
+            { to: '/stake', label: 'Stake' },
+            { to: '/moreInformation', label: 'More information' },
+          ]}
+        >
+          <Button variant="medium" appearance="primary-ghost" size="medium">
+            Blox Balance
+          </Button>
+
+          <Button
+            appearance="primary-ghost"
+            variant="medium"
+            size="medium"
+            icon="metamask"
+          />
+          <Button
+            appearance="primary-ghost"
+            variant="medium"
+            size="medium"
+            icon="polkadot"
+          />
+        </PageLayout.Header>,
+      )
+    } else {
+      setHeader(null)
+    }
+  }, [args.header, setHeader])
+
+  const tokens = ['MOVR', 'KAR', 'aUSD', 'KSM']
+  const chains = ['Moonriver', 'Karura', 'Kusama']
+
+  const [closeMenu, setCloseMenu] = useState(false)
+  const [selectedToken, setSelectedToken] = useState(tokens[0])
+  const [selectedOrigin, setSelectedOrigin] = useState(chains[0])
+  const [selectedDestination, setSelectedDestination] = useState(chains[1])
+
+  const handleClose = () => {
+    setCloseMenu(false)
+  }
+
+  const columns = {
+    asset: {
+      header: '',
+      Cell: ({ data: { icon, name } }: any) => (
+        <ColumnLayout gap={1.5}>
+          <Icon size={40} name={icon} /> <Text type="body 2">{name}</Text>
```

Comment:
Texts in both columns can have `as="span"` and that will reduce the unnecessary spacing (other that the design specified 20px)

See the first row:
![image](https://user-images.githubusercontent.com/22115870/192624827-04df6f33-ef9c-45ac-a226-1a6e38705561.png)